### ISSUE TITLE
Trouble binding MK2 puller to partial blocks(?) outside of the Overworld

### ISSUE THREAD

**Issue by MuteTiefling** [desht/ModularRouters#185]
### Describe the bug

When in the Nether, for example, it's not possible to bind an MK2 Puller to certain blocks.

I'm not currently certain what the criteria are for blocks it won't bind to, but I can recreate the issue the following: 

- Entangled
- Occultism

### How to reproduce the bug

In the Nether
1. Place a Router
2. Place a Barrel
3. Place an Entangled Block
4. With an Entangled Binder in hand, Sneak Right Click the Barrel then Right click the Entangled Block. It is now bound to the Barrel
5. Sneak Right Click the Entangled Block with an MK2 Puller. It will make a bind/unbind sound and state that it has been cleared

In the Overworld, repeat all of the above and the MK2 Puller will bind to the Entangled Block without issue.
An MK 1 Puller can pull from the Entangled Block without issue in both dimensions

You can do similar with the Occultism Dimensional Storage Actuator and Stable Wormhole as well. 
1. Place the Actuator in the Overworld (Chunk load with whatever)
2. Sneak Right Click it with a Wormhole to bind it.
3. Place the Wormhole in the Nether. 
4. MK2 Puller won't bind to the wormhole, but an MK1 can pull from it. 

The Occultism setup works fine if the Actuator is in the same dimension, oddly, unlike the Entangled Block setup.

### Expected behavior

Puller should bind to the block as it is an inventory.

### Additional details

Modular Routers: modular-routers-1.19.2-10.2.0-3
Occultism: occultism-1.19.2-1.73.2
Entangled: entangled-1.3.13-forge-mc1.19

This is for 1.19, tracker won't let me select it. 

### Which Minecraft version are you using?

1.18

**Comment by MuteTiefling** [comment]
All testing done with limited set of mods. Only JEI, Routers, Occultism, FTB Chunks, and Entangled Block

**Comment by desht** [comment]
That's a weird one, binding to blocks shouldn't be dimension-dependent like that.  Are there any errors in the log when you try to bind the puller mk2?

**Comment by MuteTiefling** [comment]
I'm not seeing any error there, no. I get a message logged when it links properly to a local storage, but when I attempt to link to a cross dim storage, I just don't see any message at all in the logs.

**Comment by desht** [comment]
Sorry, been a bit slow on this one.  I just tested in my dev instance with a router/barrel/entangled block combo in the nether, and it worked fine.  The sender visually sends items to the Entangled Block, but the items turn up in the barrel, as expected.  I haven't tested with Occultism yet but I'll try to do that today.

Some other mod interfering, maybe?

**Comment by MuteTiefling** [comment]
I'll test again later with entangled. You had it set to allow cross dim transport? I think that's off by default and it seems like the cross dimension access is the heart of the issue.

I did try again recently with updated versions of these mods though and did still see the issue with occultism. Didn't test entangled again at that time though.

My test instance only had those mods though (plus ftb chunks for chunk loading). Nothing else. Trying to strip it down to the absolute bare minimum.

**Comment by desht** [comment]
Cross dim transport?  Is that for Entangled or Occultism?  I've only tested with Entangled so far, didn't see any such option there.  But your description for Entangled didn't mention anything about cross-dimension - both barrel and entangled block are in the Nether.

**Comment by MuteTiefling** [comment]
Right. True. Alright, I'll re-test this all right now and try to clarify.

**Comment by MuteTiefling** [comment]
Ok, yes. Ignore me about cross dimensional Entangled Blocks. The initial report is accurate.

1. Place Barrel
2. Place Entangled Block
3. Bind Entangled Block to Barrel
4. Attempt to bind MK2 Sender to Entangled Block.

It is binding/unbinding on a single sneak right click.

https://streamable.com/1fjgof

For Occultism:
1. Place Dimensional Storage Actuator in Overworld
2. Chunk Load it
3. Bind a Stable Wormhole to it (sneak right click)
4. Go to the Nether
5. Place the Wormhole
6. Open it to validate it works. Place items in, for example.
7. Attempt to bind an MK2 sender to the Wormhole. 

It will bind/unbind in the single click as above. 

If the actuator is in the Nether, however, binding to a Wormhole bound to it works without issue. 
Similarly the MK2 Sender works fine if it is bound to the block space of the Wormhole prior to placing the wormhole. Place a barrel, bind, break barrel, place wormhole. 

https://streamable.com/ryxolt


Full list of mods. Mostly just libraries for the mods required to reproduce this: 

- Occultism
- Routers
- Entangled
- JEI
- FTB Chunks

![image](https://user-images.githubusercontent.com/9543430/234340298-5cc51179-043d-4b69-9671-ad0a719c8a8b.png)

[latest.log](https://github.com/desht/ModularRouters/files/11324717/latest.log)
